fxa-auth-db-server/docs/API.md

```diff
@@ -667,7 +668,8 @@ curl \
       "type": "mobile"
       "createdAt": 1437992394186,
       "callbackURL": "https://updates.push.services.mozilla.com/update/abcdef01234567890abcdefabcdef01234567890abcdef",
-      "callbackPublicKey": "468601214f60f4828b6cd5d51d9d99d212e7c73657978955f0f5a5b7e2fa1370"
+      "callbackPublicKey": "BCp93zru09_hab2Bg37LpTNG__Pw6eMPEP2hrQpwuytoj3h4chXpGc-3qqdKyqjuvAiEupsnOd_RLyc7erJHWgA",
```

Comment:
The [google web-push encryption documentation](https://developers.google.com/web/updates/2016/03/web-push-encryption?hl=en) is pretty complete.